src/main/resources/sql/insertTimeSeries.sql

```diff
@@ -24,14 +24,14 @@ insert into
 values (
 	?
 	,?
-	,NULL /*(select monitoring_location_id from monitoring_location where monitoring_location_identifier = ?)*/
+	,(select monitoring_location_id from monitoring_location where monitoring_location_identifier = ?)
 	,?
 	,?
-	,NULL
-	,NULL
+	,(select parm_nm from parm where parm_cd = ?)
```

Comment:
Not sure these are correct, the references should be a URL? (both parm & stat).

See example in https://internal.cida.usgs.gov/jira/browse/IOW-135

We may also need to do this insert differently if performance becomes as issue. Selecting twice from the same table is never as performant as doing it once. (Or in the case of the current query twice per inserted row.)

Reply:
I'm not sure about a url - there do not appear to be urls in any column in either the stat or parm tables... should we be creating these?

Reply:
Ah, I think I see what you mean.  We can concat the url "https://waterdata.usgs.gov/nwisweb/rdf?parmCd=" with the parm code for observed property reference, and we can concat "https://waterdata.usgs.gov/nwisweb/rdf?statCd=" with the stat code for statistic reference.